chapter_recurrent-neural-networks/rnn-concise.md

```diff
@@ -235,11 +235,13 @@ class RNNModel(tf.keras.layers.Layer):
         self.rnn = rnn_layer
         self.vocab_size = vocab_size
         self.dense = tf.keras.layers.Dense(vocab_size)
-    def call(self, inputs, state, _):
+
+    def call(self, inputs, state):
         X = tf.one_hot(tf.transpose(inputs), self.vocab_size)
-        Y,state = self.rnn(X, state)
+        Y, *state = self.rnn(X, state)
```

Comment:
Perhaps add a comment here on why the wildcard is needed?

Reply:
Done